modules/tournament/src/main/Scheduler.scala

```diff
@@ -98,6 +98,15 @@ private[tournament] final class Scheduler(api: TournamentApi) extends Actor {
           Schedule(Nightly, Classical, Standard, std, at(today, 9) |> orTomorrow)
         ),
 
+        List( // nightly variant tournaments!
+          Schedule(Nightly, Blitz, Chess960, std, at(today, 10) |> orTomorrow),
+          Schedule(Nightly, Blitz, KingOfTheHill, std, at(today, 11) |> orTomorrow),
+          Schedule(Nightly, Blitz, ThreeCheck, std, at(today, 12) |> orTomorrow),
+          Schedule(Nightly, Blitz, Antichess, std, at(today, 13) |> orTomorrow),
+          Schedule(Nightly, Blitz, Atomic, std, at(today, 14) |> orTomorrow),
+          Schedule(Nightly, Blitz, Horde, std, at(today, 15) |> orTomorrow)
+        ),
+
```

Comment:
not so nightly anymore according to server time. Maybe rename all nightlies to dailies? Nobody seemed pissed that there were 2 hourly bullets per hour.